shared/source/os_interface/windows/wddm/wddm.cpp

```diff
@@ -1091,4 +1090,21 @@ void Wddm::createPagingFenceLogger() {
         residencyLogger = std::make_unique<WddmResidencyLogger>(device, pagingFenceAddress);
     }
 }
+
+PhysicalDevicePciBusInfo Wddm::getPciBusInfo() const {
+    D3DKMT_ADAPTERADDRESS adapterAddress;
+    D3DKMT_QUERYADAPTERINFO queryAdapterInfo;
+
+    queryAdapterInfo.hAdapter = getAdapter();
+    queryAdapterInfo.Type = KMTQAITYPE_ADAPTERADDRESS;
+    queryAdapterInfo.pPrivateDriverData = &adapterAddress;
+    queryAdapterInfo.PrivateDriverDataSize = sizeof(adapterAddress);
+
+    if (getGdi() && getGdi()->queryAdapterInfo(&queryAdapterInfo) == STATUS_SUCCESS) {
+        return PhysicalDevicePciBusInfo(adapterAddress.BusNumber, adapterAddress.DeviceNumber, adapterAddress.FunctionNumber);
+    }
+
+    return PhysicalDevicePciBusInfo();
+}
+
```

Comment:
Missing ULT for this Wddm call. Also I don't think we need this check:
if(getGdi() )
But we may verify queryAdapterInfo(&queryAdapterInfo) == STATUS_SUCCESS

Reply:
The tests have been added. Please see [opencl/test/unit_test/os_interface/windows/wddm20_tests.cpp](https://github.com/intel/compute-runtime/pull/374/files#diff-f573679cd8a7e29aa023223be83a9eda1ab2885988052348f46ded5ef20beb1e)